erpnext/manufacturing/doctype/work_order/work_order.json

```diff
@@ -479,24 +480,28 @@
    "fieldtype": "Float",
    "label": "Raw Material Quantity"
   },
-  {
-   "default": "0",
-   "depends_on": "eval:doc.manufacturing_type==\"Process\"",
-   "fieldname": "skip_raw_material_validation",
-   "fieldtype": "Check",
-   "label": "Skip Raw Material Validation"
-  },
   {
    "fieldname": "material_request_plan_item",
    "fieldtype": "Data",
    "label": "Material Request Plan Item"
+  },
+  {
+   "fieldname": "purchase_receipt_no",
```

Comment:
this must be purchase_receipt. We always try to keep the filename and the label the same